### PR TITLE
Update Node Info message

### PR DESCRIPTION
**Pull request by lvrach**

### CODE REVIEW

File: proto/nodeinstance/info/v1/info.proto [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import "google/protobuf/timestamp.proto";
+
+// UpdateNodeInfo (Command)
+//
+// pulsar topic: `UpdateNodeInfo` (sharded)
+//
+// key: `claim_id`
+//
+// Publishers: `agent-ctrl-service`
+// Subscribers: `Node Service`
```
Comment (chrikar) [inline proto/nodeinstance/info/v1/info.proto:15]: We could also add the `cloud-node-mqtt-output-service` here.

File: proto/nodeinstance/info/v1/info.proto [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+  // virtualization_type example: kvm (optional)
+  string virtualization_type = 15;
+
+  // container_type example: docker (optional)
+  string container_type = 16;
+
+  string custom_info = 17;
+
+  map<string, string> host_labels = 21;
```
Comment (chrikar) [inline proto/nodeinstance/info/v1/info.proto:81]: [nitpick] why jump from `17` to `21`?

**Review by chrikar — APPROVED** [review]
lgtm 👍🏼 

left a couple of minor comments

**Review by nktsitas — APPROVED** [review]

File: proto/nodeinstance/info/v1/info.proto (hunk 2 above)
Comment (lvrach) [reply proto/nodeinstance/info/v1/info.proto:81]: Good catch, I have deleted the fields in between and keep the numbers to maintain compatibility with https://github.com/netdata/cloud-schemas/blob/master/proto/node/metadata/info/v1/info_updated.proto#L85.

I'll put back the deleted fields as a comment, so we don't accidentally reuse the numbers

Comment (lvrach) [reply proto/nodeinstance/info/v1/info.proto:81]: oh, 18, 19 were placed at the beginning of the proto message.

**Pull request merged**